New package: Lingeling v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: Lingeling
- Repository: https://github.com/sjvie/Lingeling.jl
- Created by: @sjvie
- Version: v1.0.0
- Commit: 36dea3916041fe86bda902ea675da667f1a387b5
- Reviewed by: @sjvie
- Reference: https://github.com/sjvie/Lingeling.jl/commit/36dea3916041fe86bda902ea675da667f1a387b5#commitcomment-152121420
- Description: Julia interface for the SAT solver Lingeling
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd66b551ec454c1c53b5ba68845e4b90772fea0691c16a9332d4b1b30148acc327bcb8022de13cbd50657fc4ce055f297c7a56a8e40cdebdabca4241617ec1e57cdc3a98786b664244547a4237fc0ed63b5e1000d159890c2309337d76f9309d36d2038e5efe39128f7910fa8887aaa6fbd574985de47724250ef82c4d4dc9056af7d56ab32fb195e46eae90c01a5e0b43fabda869ff76570059c7940f2f4b64f5359de29c1d81f9407eca72e488e5d876b -->

goerz commented:
[noblock] Are you sure you want to tag this with an initial version 1.0?

> Some documentation can be found in [Lingeling.jl](https://github.com/sjvie/Lingeling.jl/blob/9f7aa366814f36fa8319c48ad1946d73a9e3545a/src/Lingeling.jl), the Lingeling [header file](https://github.com/arminbiere/lingeling/blob/89a167d0d2efe98d983c87b5b84175b40ea55842/lglib.h), as well as in the [PicoSAT](https://fmv.jku.at/picosat/) header file.

If you are following [semantic versioning](https://semver.org/spec/v2.0.0-rc.2.html), version 1.0 indicates that a package has a stable (documented!) API. I’m not quite sure the package currently has an explicit enough documentation that would make it clear to a user what the full API is they can rely on according to the semver guarantees.

So I would recommend either improving the documentation a little, or else registering at version 0.1, and the going to 1.0 when you feel the package and documentation is really ready for the “stable” label.

JuliaRegistrator committed:
New package: Lingeling v1.0.0

UUID: 24b7b615-05f5-4cb5-9c13-cd423f90764a
Repo: https://github.com/sjvie/Lingeling.jl.git
Tree: 4ad7ebac78210a125eaee52349a9609179b34364

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

sjvie commented:
[noblock] @goerz thank's for your comment! I have updated the docs to provide more information and be more consistent overall.
